### PR TITLE
Replace `@storybook/addon-knobs` with `@storybook/addon-controls`

### PR DESCRIPTION
**Pull request by johnnyomair**

### CODE REVIEW

File: storybook/.storybook/preview.tsx [hunk 1]
```diff
@@ -4,8 +4,7 @@ import { MainContent, MuiThemeProvider } from "@comet/admin";
 import { DateFnsLocaleProvider } from "@comet/admin-date-time";
 import { createCometTheme } from "@comet/admin-theme";
 import { createTheme as createMuiTheme, GlobalStyles } from "@mui/material";
-import { select, withKnobs } from "@storybook/addon-knobs";
-import { addDecorator, addParameters } from "@storybook/react";
+import {  addParameters, ArgTypes, DecoratorFn } from "@storybook/react";
```
Comment (jamesricky) [inline storybook/.storybook/preview.tsx:7]:
```suggestion
import { addParameters, ArgTypes, DecoratorFn } from "@storybook/react";
```

File: storybook/src/admin-rte/MaxListLevel.tsx [hunk 2]
```diff
@@ -9,13 +8,29 @@ const [useRteApi] = makeRteApi();
...
     title: "@comet/admin-rte",
+    args: {
+        listLevelMax: 4,
+    },
+    argTypes: {
+        listLevelMax: {
+            name: "Max list level",
+            control: {
+                type: "number",
+                min: 1,
```
Comment (jamesricky) [inline storybook/src/admin-rte/MaxListLevel.tsx:19]: Was this changed to `1` from `0` on purpose?

Comment (johnnyomair) [reply storybook/src/admin-rte/MaxListLevel.tsx]: No, thanks. Fixed: 5595c71e000968980c4c78b52c3b9d3307ec2869

File: storybook/.storybook/preview.tsx (hunk 1 above)
Comment (johnnyomair) [reply storybook/.storybook/preview.tsx]: I added a lint for the .storybook/ folder to fix this: 27282a59ed0a825959ce261989cc93a8ec726bab

**Review by jamesricky — APPROVED** [review]

**Pull request merged**